server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateCleanupManager.java

```diff
@@ -306,7 +306,7 @@ void deleteClusterMetadata(
                     staleEphemeralAttributePaths.add(clusterMetadataManifest.getHashesOfConsistentSettings().getUploadedFilename());
                 }
                 if (clusterMetadataManifest.getClusterStateCustomMap() != null) {
-                    clusterMetadataManifest.getCustomMetadataMap()
```

Comment:
where is the custom at the metadata level getting cleaned?

Reply:
Here:
https://github.com/opensearch-project/OpenSearch/blob/dccd40f8455b3e1bc39a3f3633d9ebced5c108de/server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateCleanupManager.java#L252-L257